### PR TITLE
List create events

### PR DESCRIPTION
**Pull request by neodmy**

### CODE REVIEW

File: src/context/events-context.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    inputDescription,
+    inputToken
+  ) => {
+    const fetchedEvent = await createEventRequest(
+      inputTitle,
+      inputPrice,
+      inputDate,
+      inputDescription,
+      inputToken
+    );
```
Comment (w3dani) [inline src/context/events-context.js]: 5 params to a method is in the limit, I'd use a destructured object o something like that

File: src/pages/Events.css [hunk 2]
```diff
@@ -5,4 +5,18 @@
...
+    margin: 2rem auto;
+    list-style: none;
+    padding: 0;
+}
+
+.events__list-item {
+    margin: 1rem 0;
+    padding: 1rem;
+    border: 1px solid #463F3A;
+}
```
Comment (w3dani) [inline src/pages/Events.css:22]: You should switch to sass or scss

File: src/pages/Events.js [hunk 3]
```diff
@@ -16,8 +26,22 @@ const EventsPage = () => {
     setCreating(false);
   };
 
-  const modalConfirmHandler = () => {
+  const modalConfirmHandler = async () => {
```
Comment (w3dani) [inline src/pages/Events.js]: why async?

Comment (neodmy) [reply src/pages/Events.js]: I added it and forgot to remove 🤦

**Pull request merged**